src/services/user.ts

```diff
@@ -256,12 +255,14 @@ const setHomePhonePrivacy = (makePrivate: boolean) =>
 const setImagePrivacy = (makePrivate: boolean) =>
   http.put('profiles/image_privacy/' + (makePrivate ? 'N' : 'Y')); // 'Y' = show image, 'N' = don't show image
 
-const getBirthdate = async (): Promise<DateTime> =>
-  DateTime.fromISO(await http.get('profiles/birthdate'));
+const getBirthdate = async (): Promise<Date> => await http.get('profiles/birthdate');
```

Comment:
`http.get` returns a string, but you have typed it as returning a Date, which could cause bugs down the line. I think it would be better to convert the string to a proper Date here.

Also, when you await a promise and then immediately return the value of the promise, you can instead remove both `await` and the `async` keyword and just return the original promise directly. As this is currently written, it awaits the promise, unwraps it, and then wraps in another promise. If we get rid of both async and await, it simply returns the original promise.
```suggestion
const getBirthdate = (): Promise<Date> => http.get<string>('profiles/birthdate').then((birthdate) => new Date(birthdate));
```